data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/LogstashConfigConverter.java

```diff
@@ -0,0 +1,61 @@
+package org.opensearch.dataprepper.logstash;
+
+import com.amazon.dataprepper.model.configuration.PipelineModel;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.antlr.v4.runtime.tree.ParseTree;
+import org.opensearch.dataprepper.logstash.mapping.LogstashMapper;
+import org.opensearch.dataprepper.logstash.model.LogstashConfiguration;
+import org.opensearch.dataprepper.logstash.parser.LogstashVisitor;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Converts Logstash configuration file and returns YAML file location
+ *
+ * @since 1.2
+ */
+public class LogstashConfigConverter {
+    public String convertLogstashConfigurationToPipeline(String logstashConfigurationPath, String outputDirectory) throws IOException {
+        Path configurationFilePath = Paths.get(logstashConfigurationPath);
+        final String logstashConfigAsString = new String(Files.readAllBytes(configurationFilePath));
+
+        LogstashLexer lexer = new LogstashLexer(CharStreams.fromString(logstashConfigAsString));
+        CommonTokenStream tokens = new CommonTokenStream(lexer);
+        LogstashParser parser = new LogstashParser(tokens);
+        ParseTree tree = parser.config();
+
+        org.opensearch.dataprepper.logstash.parser.LogstashVisitor visitor = new LogstashVisitor();
```

Comment:
You need to have the fully qualified name on the right-side of the assignment, along with the `new`.
```
org.opensearch.dataprepper.logstash.parser.LogstashVisitor visitor = new org.opensearch.dataprepper.logstash.parser.LogstashVisitor();
```